src/groupCurrencyToken.ts

```diff
@@ -114,3 +118,15 @@ export function handleSuspended(event: SuspendedEvent): void {
   groupCurrencyToken.suspended = GroupCurrencyTokenContractInstance.suspended()
   groupCurrencyToken.save()
 }
+
+// @TODO: this event has many information such as: receiver, amount, mintAmount, mintFee
+// amount = mintAmount + mintFee
+// we might want to save some of this piece of information in a different entity?
+// in the meantime we will just save the minted amount (mintAmount)
```

Comment:
Do you think it would make sense to also store the accumulated mint fees?

Reply:
I think It would make sense to save everything (mintAmount + mintFee) but I wasn't sure to save both in the same property, I could add a new property to save the accumulated minted fees

Reply:
I'll merge it and work on this in a separate issue